crates/api/api_crud/src/multi_community/create.rs

```diff
@@ -51,5 +51,8 @@ pub async fn create_multi_community(
     )
   };
   let multi = MultiCommunity::create(&mut context.pool(), &form).await?;
-  get_multi(multi.id, context).await
+
+  // TODO what about following your own community?
```

Comment:
Main question here. I don't see any following of the communities you've created, which should be the default.

Reply:
Makes sense this should be added.